ocs_ci/deployment/aws.py

```diff
@@ -100,16 +112,18 @@ def deploy(self, log_cli_level='DEBUG'):
                 log_cli_level (str): openshift installer's log level
                     (default: "DEBUG")
             """
-            logger.info("Deploying OCP cluster")
-            logger.info(
-                f"Openshift-installer will be using loglevel:{log_cli_level}"
-            )
-            run_cmd(
-                f"{self.installer} create cluster "
-                f"--dir {self.cluster_path} "
-                f"--log-level {log_cli_level}"
-            )
-            self.test_cluster()
+            aws_ipi = AWSIPI()
```

Comment:
Do not create another object of AWSIPI deployment here in OCP deployment. Rather do this check in deploy_ocp method right after line 135 here: https://github.com/red-hat-storage/ocs-ci/pull/780/files#diff-9706766d7ecd5b59c5911c747824fb53R135

In case the cluster exist just raise the exception and let the deployment fail there.

Reply:
done